src/main/resources/scripts/kill counter.sk

```diff
@@ -5,22 +5,22 @@
 # Defaults are useful here, as a player's kills would be '<none>' if he hasn't killed anything yet,
 # but with defaults defined those will be used in that case.
 variables:
-	{kill counter.%player%.kills_session} = 0
-	{kill counter.%player%.kills_session_max} = 0
-	{kill counter.%player%.kills_total} = 0
+	{kill counter::%player%::kills_session} = 0
```

Comment:
Player names in variables is bad practice. Preferrably use UUIDs.

Reply:
Can't here, that's the reason why I didn't use it here. Variables doesn't work with ``uuid of player``.

Reply:
They do. Variables don't work with the uuid of a player **variable**.

Reply:
After testing, confirmed that `Variables` doesn't work with uuid of player, but in that case, the usage of the Default Value expression would be preferred.

Reply:
I think you can just use %string%, since the UUID is a string in Skript.